hermes landing page (#1515)
getsentry/sentry-docs: src/collections/_documentation/clients/react-native/hermes.md

---
title: 'Hermes'
robots: noindex
---

Sentry added support to `hermes` which required changes to the SDK as well as Sentry itself.

> Sentry customers, using the SaaS offer (sentry.io), only need to update the SDK.

Reviewer: ```suggestion
Sentry customers using the SaaS product (sentry.io) will only need to update the SDK.
```

> The minimum required version for the SDK is `sentry-react-native` [SDK version `1.3.3`](https://github.com/getsentry/sentry-react-native/releases/tag/1.3.0).

Reviewer 2: ```suggestion
The minimum required version for the SDK is `sentry-react-native` [SDK version `1.3.3`](https://github.com/getsentry/sentry-react-native/releases/tag/1.3.3).
```

> The minimum required version for the SDK is `sentry-react-native` [SDK version `1.3.3`](https://github.com/getsentry/sentry-react-native/releases/tag/1.3.3).

Reviewer 3: ```suggestion
The minimum required version for the SDK is `@sentry/react-native` [SDK version `1.3.3`](https://github.com/getsentry/sentry-react-native/releases/tag/1.3.3).
```
